armi/bookkeeping/mainInterface.py

```diff
@@ -165,6 +165,8 @@ def interactEOL(self):
 
     def updateClusterProgress(self):
         """Updates the status window on the Windows HPC client."""
+        if not shutil.which("job"):
+            return
```

Comment:
Did this make it in by accident? System-specific

Reply:
nope. It was throwing errors on systems that don't have job so this suppresses that.

Reply:
Holy cow, forest for the trees! Didn't see that this was in an `updateClusterProgress()` function. We need to get this into the cluster plugin. Adding this line is good for now though.